tests/postgres_tests/test_indexes.py

```diff
@@ -1,8 +1,32 @@
-from django.contrib.postgres.indexes import GinIndex
+from django.contrib.postgres.indexes import BrinIndex, GinIndex
 from django.db import connection
+from django.test import skipUnlessDBFeature
 
 from . import PostgreSQLTestCase
-from .models import IntegerArrayModel
+from .models import CharFieldModel, IntegerArrayModel
+
+
+@skipUnlessDBFeature('has_brin_index_support')
+class BRINIndexTests(PostgreSQLTestCase):
+
+    def test_eq(self):
```

Comment:
If a custom `__eq__()` method isn't necesary, then it seems like these tests aren't needed.